AppAuth-JS/built/crypto_utils.d.ts

```diff
@@ -0,0 +1,3 @@
+export declare function bufferToString(buffer: Uint8Array): string;
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork